stl/inc/variant

```diff
@@ -39,6 +39,12 @@ struct _All_same : true_type {};
 template <class _First, class... _Rest>
 struct _All_same<_First, _Rest...> : bool_constant<conjunction_v<is_same<_First, _Rest>...>> {}; // variadic is_same
 
+template <class...>
+struct _All_convertible : true_type {};
+template <class _To, class... _From>
+struct _All_convertible<_To, _From...> : bool_constant<conjunction_v<is_convertible<_From, _To>...>> {

```

Comment:
Naming: I am concerned that this reverses the `_From, _To` order. The reversal is necessary because of the variadic `_From`, but the name doesn't draw attention to that fact. (In metaprogramming, especially pre-concepts, we don't really have a type system to enforce proper usage, so conventions and naming are extremely important.)

Idea: `_Convertible_to_X_from_all`, or something else?